README.md

```
@@ -1 +1,75 @@
+		string title
+		string body
+		datetime created_at
+	}
+	
+	VOLUNTEER ||--o{ VOLUNTEER_RECORD : AssignVolunteerHours
+	USER ||--o{ VOLUNTEER_RECORD : AssignVolunteerHours
+	VOLUNTEER ||--o{ PARTICIPATION : application
+	USER ||--o{ PARTICIPATION : application
+	VOLUNTEER ||--o{ ACTIVITY : VolunteerDetail
+	USER_ACTIVITY }o--|| ACTIVITY : TeacherArea
+	USER ||--o{ USER_ACTIVITY : asdf
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)